cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -390,7 +394,8 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
     testQuery(
       title = "Drop a non-existing constraint",
       text = "If it is uncertain if any constraint with a given name exists and you want to drop it if it does but not get an error should it not, use `IF EXISTS`. " +
-        "It is the same command for unique property, property existence and node key constraints.",
+        "It is the same command for unique property, property existence and node key constraints. " +
+        "Note: The `IF [NOT] EXISTS` syntax for constraints is only available in Neo4j 4.1.3 and onwards.",
```

Comment:
Since this is `DROP` the `NOT` should not exist
```suggestion
        "Note: The `IF EXISTS` syntax for constraints is only available in Neo4j 4.1.3 and onwards.",
```